docs/circuits.md

```diff
@@ -362,3 +362,109 @@ print(circuit[1:3])
+    include "qelib1.inc";
+    qreg q[3];
+    creg meas[3];
+    h q;
+    measure q -> meas;
+    """)
+print(circuit)
+```
+
+Results: 
+
+```
+q_0: ───H───M('meas_0')───
+
+q_1: ───H───M('meas_1')───
+
+q_2: ───H───M('meas_2')───
+```
+
+
+#### Supported Control statements 
+
+
+| Statement|Cirq support|Description|Example|
+|----| --------| --------| --------|
+| `OPENQASM 2.0;`| supported| Denotes a file in Open QASM format| OPENQASM 2.0;|
+| `qreg name[size];`| supported (see mapping qubits)| Declare a named register of qubits| `qreg q[5]; `|
+| `creg name[size];`|supported (see mapping classical register to measurement keys)|  Declare a named register of bits| `creg c[5];`|
+| `include “filename”; `| supported ONLY to include the standard “qelib1.inc” lib for compatibility| Open and parse another source file| `include “qelib1.inc”;`|
+| `gate name(params) qargs;`|NOT supported| Declare a unitary gate||
+|`opaque name(params) qargs;`| NOT supported| Declare an opaque gate||
+|  `// comment text`| supported|Comment a line of text| `// supported!`|
+|`U(theta,phi,lambda) qubit/qreg;`|  supported| Apply built-in single qubit gate(s)| `U(pi/2,2\*pi/3,0) q[0];`|
+|  `CX qubit/qreg,qubit\qreg;`| supported|Apply built-in CNOT gate(s)| `CX q[0],q[1];`|
+| ` measure qubit\qreg -> bit/creg;`| supported|Make measurement(s) in `Z` basis| `measure q -> c;`|
+|  `reset qubit/qreg;`| NOT supported|Prepare qubit(s) in`|0>`| `reset q[0]; `| 
+| `gatename(params) qargs;`|  supported for ONLY the supported subset of standard gates defined in "qelib1.inc"|Apply a user-defined unitary gate| `rz(pi/2) q[0];`|
+| `if(creg==int) qop;`| NOT supported| Conditionally apply quantum operation| `if(c==5) CX q[0],q[1];`|
+|  `barrier qargs;`| NOT supported| Prevent transformations across this source line| `barrier q[0],q[1];`|
+
+
+#### Gate conversion rules 
+
+Note: The standard Quantum Experience standard gates are defined in ([qelib.inc](https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/qasm/libs/qelib1.inc)) 
+based on the `U` and `CX` built-in instructions and we could generate them dynamically. Instead, we chose to map them to native cirq gates explicitly which results in a more user friendly circuit. 
+
+| QE gates| Cirq translation| Notes|
+| --------| --------| --------|
+| u3(θ,φ,λ)| `QasmUGate (λ,θ,φ)`| `QasmUGate` constructor takes angles reverse order, to cater for the rotation intuition RZ(λ) first, RY(θ), RZ(φ) last|
+|u2(φ,λ) = u3(π/2,φ,λ)| `QasmUGate (λ,π/2,φ)`|| 
+|u1 (φ) = u3(0,0,λ)| NOT supported || 
+| cx| `cirq.CX`|| 
+| id| `cirq.Identity`| one single-qubit Identity gate is created for each qubit if applied on a register|  
+| u0(γ)| NOT supported| this is the "WAIT gate" for length γ in QE| 
+| x| `cirq.X`|| 
+| y| `cirq.Y`|| 
+| z| `cirq.Z`|| 
+| h| `cirq.H`|| 
+| s| `cirq.S`|| 
+| sdg| `cirq.S**-1`|| 
+| t| `cirq.T`|| 
+| tdg| `cirq.T**-1`||
+| rx(theta)| `cirq.RX`|| 
+| ry(theta)| `cirq.RY`|| 
+| rz(phi)|`cirq.RZ`|| 
```

Comment:
The three rotation gates are actually `cirq.Rx(theta)`, `cirq.Ry(theta)`, `cirq.Rz(phi)` (`Rx`, `Ry`, and `Rz` are actually functions that return gates).